src/domain/ViewModel.ts

```diff
@@ -58,6 +58,14 @@ export class ViewModel<O extends Options = Options> extends EventEmitter<{change
         return this._options[name];
     }
 
+    observeNavigation(type: string, onChange: (value: string | true | undefined, type: string) => void) {
```

Comment:
Once #676 is merged, we can make the type an value type-safe here, but this works for now :+1: